buildSrc/src/main/java/dependencies/Dependencies.kt

```diff
@@ -11,39 +11,39 @@ object Dependencies {
     const val androidx_lifecycle_livedata_ktx = "androidx.lifecycle:lifecycle-livedata-ktx:2.3.1" // Upgrading will require compileSdk 31
     const val androidx_lifecycle_viewmodel = "androidx.lifecycle:lifecycle-viewmodel:2.3.1" // Upgrading will require compileSdk 31
     const val androidx_core_ktx = "androidx.core:core-ktx:1.6.0" // Upgrading will require compileSdk 31
-    const val androidx_browser = "androidx.browser:browser:1.3.0"
+    const val androidx_browser = "androidx.browser:browser:1.3.0" // Upgrading will require compileSdk 31
```

Comment:
Can we see if we can get away with this now? If it's simple I don't think there's a reason not to do it.

Reply:
You mean bumping compileSdk to 31 as a part of this pr?

Reply:
Yeah, or at least just investigating if that's something we can't do. Just looks like there are a lot of `androidx` dependencies that will fall behind otherwise.